package.json

```diff
@@ -14,7 +14,7 @@
     "@grafana/data": "7.1.1",
     "@grafana/e2e": "7.1.1",
     "@grafana/eslint-config": "2.0.0",
-    "@grafana/runtime": "7.1.1",
+    "@grafana/runtime": "7.4.0",
```

Comment:
Can you also update the dependencies in plugin.json? External plugins will receive packages from the grafana runtime, so if ADX is depending on grafana 7.4, this needs to be updated.

Reply:
yes, i did :)